internal/snmp/translate.go

```diff
@@ -71,7 +75,7 @@ func LoadMibsFromPath(paths []string, log telegraf.Logger) error {
 			return nil
 		})
 		if err != nil {
-			return fmt.Errorf("Filepath could not be walked: %v", err)
+			return fmt.Errorf("Filepath %q could not be walked: %v", mibPath, err)
 		}
```

Comment:
Since the function above does not return an error now, err will always be nil.

Reply:
Ah, this is not intended. In the previous function, if we find an error which isn't `NotExist`, we should return this error.
Pushing a fix.